app/controllers/cookies_consent_controller.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+
+class CookiesConsentController < ApplicationController
+  def self.get_cookie_policy_value(cookie)
+    unless cookie.nil?
+      JSON.parse(cookie)
+    end
+  end
+
+  def accept_cookies
+    set_cookie_policy(true)
+    redirect_back(fallback_location: root_path)
+  end
+
+  def reject_cookies
+    set_cookie_policy(false)
+    redirect_back(fallback_location: root_path)
+  end
+
+  def add_seen_confirmation_message
+    set_cookie_preference
+    redirect_back(fallback_location: root_path)
+  end
+
+  private
+
+  def set_cookie_policy(cookies_allowed)
```

Comment:
This method is duplicated here https://github.com/trade-tariff/trade-tariff-frontend/pull/33/files/9d1a5aa6030797ed3e886bf5aef4112d82a85f6a#diff-d22397cc3f79c2827d23d685cdf4d04de82c72087b07825ecabb264fbe91f82fR38, the only difference seems to be the policy (usage key) - could we move this to a cookies_helper?